packages/connect/src/api/ethereum/__fixtures__/ethereumSignTx.ts

```diff
@@ -74,6 +77,7 @@ export const serializeEthereumTx: EthereumTxFixture[] = [
             r: '0x9d4599beedc587e0dc3d88578d79573c0138f9389810ffb036c37423ccd86375',
             s: '0x4a0eb870fbae9a11a02e3e0068830d141ee952bb4ab4d1e1b7542d75f7a24dc1',
         },
+        defaultHardfork: Hardfork.Petersburg,
```

Comment:
I am not familiar with this topic. Does it sound right? @martykan

Reply:
Yeah should be right, we already had this configured before for ETC